lib/model/model.go

```diff
@@ -223,7 +233,7 @@ type ConnectionInfo struct {
 	Paused        bool
 	Address       string
 	ClientVersion string
-	Type          ConnectionType
+	Direct        bool
```

Comment:
And you can have it, but at the moment this is the only aspect of the connection we actually care about in here.